docs/source/models/gbd2017/causes/tb_hiv_model_vv.rst

```diff
@@ -0,0 +1,87 @@
+.. _cause_model_validation:
+
+========================================
+TB/HIV model validation and verification
+========================================
+
+Obejective
+----------
+External
+	- Model results should be checked against local TB epidemiology
+		- e.g. rate of decline in burden of disease should be compared
+		  with historical evidence.
+	- Compare our results (e.g. ICERs) to similar models or empirical
+	  assessments where possible
+Internal
+	- Calibrate simulation baseline against GBD 2017 results
+		- Sim outputs mean should perfectly match GBD results
+		- Sim outputs uncertainty should wider than GBD results
+			- Stochastic + individual heterogeneity
+
+logic
+-----
+- Parent cause is the sum of child causes
+	- Fatal: Deaths (CSMR, Excess MR), YLLs
+	- Non-fatal: Cases, YLDs, Prevalence, Incidence
+	- DALYs = YLLs + YLDs
+- By location-/age-/sex-
+- Start from count space
+- We expect that active TB cases < LTBI cases when LTBI prevalence
+  as denominator

```

Comment:
This confuses me.  Do you mean the incidence *rate*, since you say denominator? or do you mean the count, since only a fraction of LTBI progresses to active TB?

Reply:
I will rephrase it. Here i mean the simulation output on event count, the total LTBI cases should be greater than active TB cases (active TB cases / LTBI cases < 1).